mobile/src/profile/home-profile-screen.tsx

```diff
@@ -0,0 +1,81 @@
+import React, { Component } from 'react';
+import { connect } from 'react-redux';
+import { View, LayoutChangeEvent, Text, Image } from 'react-native';
+
+import styles from '../layout/styles';
+import { layoutStyles, contentStyles } from './styles';
+import { Chevron } from './chevron';
+import { Avatar } from '../people/avatar';
+import { TopNavBar } from '../topNavBar/top-nav-bar';
+import { AppState } from '../reducers/app.reducer';
+import { UserInfoState } from '../reducers/user/user-info.reducer';
+import { Department } from '../reducers/organization/department.model';
+
+const navBar = new TopNavBar('');
+
+interface ProfileScreenProps {
+    userInfo: UserInfoState;
+    departments: Department[];
+}
+
+const mapStateToProps = (state: AppState): ProfileScreenProps => ({
+    userInfo: state.userInfo,
+    departments: state.organization.departments
+});
+
+class ProfileScreenImpl extends Component<ProfileScreenProps> {
+    public static navigationOptions = navBar.configurate();
+
+    public render() {
+        const userInfo = this.props.userInfo;
+
+        const employee = userInfo ? userInfo.employee : null;
+        const name = employee ? employee.name : null;
+        const position = this.uppercase(employee ? employee.position : null);
```

Comment:
I keep seeing the same code over and over again. Maybe have an EmployeePosition component and EmployeeName component?

Reply:
Because of 1 line?

Reply:
check out @victor-kotov-arcadia PR, he has exact lines :) Even including uppercase. Now, if you decide to change it, you're bound to forget to change it everywhere

Reply:
Hello there?